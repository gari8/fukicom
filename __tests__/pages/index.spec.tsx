import Index from "@/pages/index";
import { render } from "@testing-library/react";

jest.mock("@/hooks/useMeta", () => ({
  useMeta: () => {
    return {
      title: "",
      metadata: [],
      favicon: "",
    };
  },
}));

describe("Index", (): void => {
  test("is components", () => {
    const { asFragment } = render(<Index />);
    expect(asFragment()).toMatchSnapshot();
  });
});
