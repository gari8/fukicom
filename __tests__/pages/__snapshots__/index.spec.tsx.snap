// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index is components 1`] = `
<DocumentFragment>
  <div
    class="css-0"
  >
    <header
      style="height: 50px; box-shadow: 0px 2px 2px rgba(0, 0, 0, 0.25);"
    >
      <img
        class="chakra-image css-16kb41"
        src="/logo.png"
      />
    </header>
    <main
      style="min-height: calc(100vh - 50px - 40px);"
    />
    <footer
      style="height: 40px; background-color: lightgray; display: flex;"
    >
      <img
        class="chakra-image css-0"
        src="/logo.png"
      />
      <p
        class="chakra-text css-0"
      />
    </footer>
  </div>
</DocumentFragment>
`;
